hack/lib.sh

```diff
@@ -266,14 +266,14 @@ check_all_deployments_ready() {
   local deployments
   deployments=$(kubectl -n $namespace get deployments -o json)
 
-  if [ $(jq '.items | length' <<< $deployments) -eq 0 ]; then
```

Comment:
IIRC, this is the way shellcheck is insisting it needs to be. Do we want to drop it?

Reply:
Fixed